multiqc/modules/mtnucratio/mtnucratio.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python
+
+""" MultiQC module to parse output from mtnucratio """
+
+from __future__ import print_function
+from collections import OrderedDict
+import logging
+import os
+import re
+import json
+
+from multiqc import config
+from multiqc.modules.base_module import BaseMultiqcModule
+
+# Initialise the logger
+log = logging.getLogger(__name__)
+
+class MultiqcModule(BaseMultiqcModule):
+    """ mtnucratio module """
+
+    def __init__(self):
+
+        # Initialise the parent object
+        super(MultiqcModule, self).__init__(name='mtnucratio', anchor='mtnucratio',
+        href="http://www.github.com/apeltzer/MTNucRatioCalculator",
+        info="is a tool to compute mt/nuc ratios for NGS datasets.")
+
+        # Find and load any MTNUCRATIO reports
+        self.mtnuc_data = dict()
+
+        for f in self.find_log_files('mtnucratio',filehandles=True):
+            self.parseJSON(f)
+
+        # Filter to strip out ignored sample names
+        self.mtnuc_data = self.ignore_samples(self.mtnuc_data)
+
+        if len(self.mtnuc_data) == 0:
+            raise UserWarning
+
+        log.info("Found {} reports".format(len(self.mtnuc_data)))
+
+        # Write parsed report data to a file
+        self.write_data_file(self.mtnuc_data, 'multiqc_mtnucratio')
+
+        # Basic Stats Table
+        self.mtnucratio_general_stats_table()
+
+
+    #Parse our nice little JSON file
+    def parseJSON(self, f):
+        """ Parse the JSON output from mtnucratio and save the summary statistics """
+        try:
+            parsed_json = json.load(f['f'])
+            if 'metrics' not in parsed_json and 'metadata' not in parsed_json:
+                log.warn("No MTNUCRATIO JSON: '{}'".format(f['fn']))
+                return None 
+        except JSONDecodeError as e:
+            log.warn("Could not parse mtnucratio JSON: '{}'".format(f['fn']))
+            print(e)
```

Comment:
Still don't really like having `print` statements (stdout instead of stderr, etc). But this should very rarely happen so should be fine. Maybe I'll just fix it myself quickly after merging.. :)